src/bin/pgcopydb/ld_stream.c

```diff
@@ -767,8 +775,49 @@ streamRotateFile(LogicalStreamContext *context)
 		return true;
 	}
 
-	/* compute the WAL filename that would host the current LSN */
-	XLByteToSeg(context->cur_record_lsn, segno, context->WalSegSz);
+	/*
+     * determine the LSN to calculate walFileName in which to write the current
+     * message.
```

Comment:
single line comments start lowercase.

Multiple lines comments are actual English sentences and start with a capital letter. I know it's nitpicking, that said if I don't share that information I just make it hard for you to get to it ;-)

Reply:
I understand the importance of this. Thank you for point this out, I'll keep this in mind.